tools/privacyidea-authorizedkeys

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# privacyidea-authorizedkeys
+#
+# This script is based on privacyidea-fetchssh and can now be used togeather with
+# the new service_id to in the authorizedkeys-command.
+#
+#
+# History
+# v0.1	03/31/2015	Peter Murr	mailinglists [at] pcfreak [dot] de
+# v0.2  2023-02-23  cornelius.koelbel@netknights.it --- use to run in SSH config
+
+
+#input parameters
+#------ Change this to your needs -----
+server='localhost:5000'
+serviceaccount='admin'
+password='test'
+service_id='webservers'
+#--------------------------------------
+hostname=`hostname`
+insecure=""
+#insecure="-k"
+
+user=$1
+
+# some checks for dependencies
+command -v jq >/dev/null 2>&1 || { echo >&2 "script requires jq but it is not installed.  Aborting."; exit 2; }
+command -v sed >/dev/null 2>&1 || { echo >&2 "script requires sed but it is not installed.  Aborting."; exit 2; }
+command -v sed >/dev/null 2>&1 || { echo >&2 "script requires curl but it is not installed.  Aborting."; exit 2; }
```

Comment:
Please check for `curl` and `tr`, `sed` is checked already...

Reply:
Done in 837ac02b